exploring-ship-logbooks/word_count.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
```

Comment:
I use the iterrows() function on the "data" data frame to scroll through the data.